README.md

```diff
@@ -49,7 +53,8 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+town[:residents] = ["Maurice", "Gaston"]
```

Comment:
While not wrong to over-write the existing array, It would be better here to use `.slice` or `.delete` to remove just the one element.